src/api/users.ts

```diff
@@ -0,0 +1,20 @@
+import { User } from '../types/User';
+import { client } from '../utils/fetchClient';
+
+export const USER_ID = 1549;
+
+export const getUsersFromServer = () => {
+  return client.get<User[]>(`/users`);
```

Comment:
The endpoint `/users` is hardcoded in the `getUsersFromServer` function. Ensure that this endpoint is correct and matches the API specification you are working with.